src/commands/rsync.ts

```diff
@@ -20,10 +42,29 @@ export const rsyncCommand = async () => {
         placeHolder: "Which plugin would you like to sync?",
     });
 
-    const wpPath = await vscode.window.showInputBox({
-        prompt: 'Enter the remote path to upload the plugin contents to.',
-        placeHolder: 'user@server:public_html/wp-content/plugins/jetpack',
-    });
+	const existingSite = await vscode.window.showQuickPick(getSavedSites(configFilePath), {
+		placeHolder: "Which site would you like to sync to?"
+	});
+
+	if (typeof existingSite === 'undefined') {
```

Comment:
Hmm, wouldn't a simple `!existingSite` be enough in this case or are there some other considerations we're taking into account here that I could be missing?